pep-8002.rst

```diff
@@ -371,6 +372,114 @@ code they deliver allows different teams to have different
 interpretations of the same requirements. For example, there are now
 several teams working on different deployment tools.
 
+
+Jupyter
+=======
+
+The governance structure is documented in the `Main Governance Document
+<https://github.com/jupyter/governance/blob/master/governance.md>`_
+within the `Jupyter Governance repo <https://github.com/jupyter/governance>`_.
+
+The effective governance process grows organically over time as the needs of
+the project evolve. Formal changes to the Governance Document are submitted via
+Pull Request, with an open period for comments. After the open period, a
+Steering Council may call for a vote to ratify the PR changes. Acceptance
+requires a minimum of 80% of the Steering Council to vote and at least 2/3 of
+the vote must be positive. The BDFL can act alone to accept or reject changes
+or override the Steering Council decision; though this would be an extremely
+rare event.
+
+Key people and their functions
+------------------------------
+
+The key people in Jupyter's Governance are the BDFL, Fernando Perez, and the
```

Comment:
I would expect a sentence explaining whether the BDFL was there from the start (is he the project founder?) and whether it's truly "for life".